### PR TITLE
Changed build locations

### PR DESCRIPTION
**Pull request by wooferzfg**
I think that it should actually build into the components folder like the other
components, but you might build it to a different place, idk.

### CODE REVIEW

**Comment by 0x0ade** [comment]
... Nobody notified me about it? Nobody noticed that the deployed .zip files now are empty?

_sigh_

Lemme fix dat in a while.

**Comment by Gyoo** [comment]
Ah, sorry about that, I completely forgot that you have an automatic export of the dev builds before merging the PR.
Partly my mistake, I guess.

**Comment by 0x0ade** [comment]
Nah, it's not that bad. It's just that I just came from school, checked my emails and saw this. I've had to "fix" the path once (it tried to create a folder on the root partition) and I don't know how long this new path will still work on Travis CI. But deploy.sh is updated now, so it shouldn't be a problem for now.

TL;DR: Worried about path breaking on Travis CI as it did before; Updated deploy.sh
